_posts/2018-10-21-kubernetes-testing-e2e.markdown

```diff
@@ -0,0 +1,320 @@
+---
+layout: post
+title:  "Testing and Kubernetes"
+date:   2018-10-21
+tags: Kubernetes Go Conformance BDD
+author: Matt Tyler
+image: img/kubernetes/k8s.png 
+---
+
+## Cliff Notes
```

Comment:
I'd never heard the term cliff notes before. Happy to keep it, just wondering if it was me or others hadn't either?